css/site.styl

```diff
@@ -88,6 +88,42 @@ img
 
 // ******************************************************
 
+// HOME
+
+.home-work-item
+  opacity: 0
+  transition: opacity .2s
+
+.home-work-contents
+  position: relative
+  font-size: 0
```

Comment:
ah it was to kill the whitespace below the iamge but i shoudl just be setting it to block level